deploy/helm/kuberhealthy/templates/khcheck-pod-restarts.yaml

```diff
@@ -42,4 +42,41 @@ spec:
 {{- toYaml .Values.check.podRestarts.nodeSelector | nindent 6 }}
     {{- end }}
     restartPolicy: Never
-{{- end }}
+    serviceAccountName: pod-restart-sa
```

Comment:
This is fine here, but in the future you don't need to label service accounts with `-sa` at the end.  It will be clear when you fetch objects of type `serviceAccount`.

Reply:
Noted, I didn't know if we wanted to stick to a convention or something.